sor/src/main/java/com/bazaarvoice/emodb/sor/db/DeltaIterator.java

```diff
@@ -194,11 +208,26 @@ private ByteBuffer stitchContent(int contentSize) {
         return content;
     }
 
-    // transforms delta from the provided type to return type
-    abstract protected T convertDelta(R delta);
+    protected static class BlockedDelta {
+        private final int _numBlocks;
+        private final ByteBuffer _content;
+
+        public BlockedDelta(int numBlocks, ByteBuffer content) {
+            _numBlocks = numBlocks;
+            _content = content;
+        }
+
+        public int getNumBlocks() {
+            return _numBlocks;
+        }
+
+        public ByteBuffer getContent() {
+            return _content;
+        }
+    }
 
     // transforms delta from the provided type to the return type, while also changing content to be the parameter provided
-    abstract protected T convertDelta(R delta, ByteBuffer content);
+    abstract protected T convertDelta(R delta, BlockedDelta blockedDelta);
 
     // get block number from delta
     abstract protected int getBlock(R delta);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)